data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -104,16 +105,20 @@ def iter_elife_manuscript_part_of_section_complement_for_one_record(
 ) -> Iterable[DocmapPartOfComplement]:
     if related_content['manuscript_id']:
         assert related_content['manuscript_type']
+        manuscript_type = related_content['manuscript_type'].lower().strip()
+        cleaned_manuscript_type = re.sub(r'[^a-z0-9 ]', ' ', manuscript_type)
```

Comment:
Maybe we don't need this cleaning step as the manuscript type wouldn't include any unnecessary characters. We can just remove this part and update the test data instead something like "manuscript type 1". Let me know what do you think.

Reply:
Have made adjustments